path_static_roles_test.go

```diff
@@ -78,116 +78,6 @@ func TestRoles(t *testing.T) {
 			t.Fatal("expected last_vault_rotation to not be empty")
 		}
 	})
-	t.Run("happy path with roles", func(t *testing.T) {
-		b, storage := getBackend(false)
-		defer b.Cleanup(context.Background())
-
-		data := map[string]interface{}{
-			"binddn":      "tester",
-			"bindpass":    "pa$$w0rd",
-			"url":         "ldap://138.91.247.105",
-			"certificate": validCertificate,
-		}
-
-		req := &logical.Request{
-			Operation: logical.CreateOperation,
-			Path:      configPath,
-			Storage:   storage,
-			Data:      data,
-		}
-
-		resp, err := b.HandleRequest(context.Background(), req)
-		if err != nil || (resp != nil && resp.IsError()) {
-			t.Fatalf("err:%s resp:%#v\n", err, resp)
-		}
-
-		data = map[string]interface{}{
-			"username":        "hashicorp",
-			"dn":              "uid=hashicorp,ou=users,dc=hashicorp,dc=com",
-			"rotation_period": "5s",
-		}
-
-		req = &logical.Request{
-			Operation: logical.CreateOperation,
-			Path:      staticRolePath + "hashicorp",
-			Storage:   storage,
-			Data:      data,
-		}
-
-		resp, err = b.HandleRequest(context.Background(), req)
-		if err != nil || (resp != nil && resp.IsError()) {
-			t.Fatalf("err:%s resp:%#v\n", err, resp)
-		}
-
-		req = &logical.Request{
-			Operation: logical.ReadOperation,
-			Path:      staticRolePath + "hashicorp",
-			Storage:   storage,
-			Data:      nil,
-		}
-
-		resp, err = b.HandleRequest(context.Background(), req)
-		if err != nil || (resp != nil && resp.IsError()) {
-			t.Fatalf("err:%s resp:%#v\n", err, resp)
-		}
-
-		if resp.Data["dn"] != data["dn"] {
-			t.Fatalf("expected dn to be %s but got %s", data["dn"], resp.Data["dn"])
-		}
-
-		if resp.Data["username"] != data["username"] {
-			t.Fatalf("expected username to be %s but got %s", data["username"], resp.Data["username"])
-		}
-
-		if resp.Data["rotation_period"] != float64(5) {
-			t.Fatalf("expected rotation_period to be %d but got %s", 5, resp.Data["rotation_period"])
-		}
-
-		if resp.Data["last_vault_rotation"] == nil {
-			t.Fatal("expected last_vault_rotation to not be empty")
-		}
-	})
-
-	t.Run("missing dn", func(t *testing.T) {
```

Comment:
Why are we removing the `missing dn` test? I see that the `happy path` test has been duplicated so that one makes sense, but wasn't clear on this one :)

Reply:
Whoops! Went a little too far when removing the duplicate test. Good catch, thanks! Added back in 1572a37.